data/wordpressa/santuario-xml-security1/misuses/SV_CVE_13_2172/misuse.yml

```diff
@@ -0,0 +1,20 @@
+api:
```

Comment:
This file is incomplete.

Reply:
This is fixed with commit 5055704a9dac770e31a7e519dff05652aead61d9.